.circleci/config.yml

```diff
@@ -334,7 +334,7 @@ jobs:
                 Import-PfxCertificate -FilePath $env:CSC_LINK -CertStoreLocation Cert:\LocalMachine\Root -Password (ConvertTo-SecureString -String $env:WIN_CSC_KEY_PASSWORD -AsPlainText -Force)
 
                 # $env:CSC_KEY_PASSWORD=$env:WIN_CSC_KEY_PASSWORD
-                make package
+                make package ELECTRON_BUILDER_ARGS='-c.win.certificateSubjectName="Automattic, Inc."'
```

Comment:
Note: This parameter is required in Circle, but will fail the build locally (unless the user installs the certificate to their certificate store). Therefore we remove it from package.json and pass it only in the Circle job, so packaging the application locally can work as expected (i.e. with the `CSC_LINK/PASSWORD` environment variables).